bundled/muparser_v2_2_4/include/muParserTokenReader.h

```diff
@@ -1,161 +1,161 @@
-/*

```

Comment:
What's going on with this file? Did you change the line ending?

Reply:
Yes. Something incorrect had happened. I have restored this file to the former state now (I think).